### PR TITLE
sync image data

### PR DESCRIPTION
**Pull request by Aychihuahuausa**

### CODE REVIEW

File: src/app/main-client/main-client.component.ts [hunk 1]
```diff
@@ -139,7 +139,7 @@ export class MainClientComponent implements OnInit {
   }
 
   ngOnInit(): void {
-
+    this.appServc.getSysConfig();
```
Comment (heshambadawy) [inline src/app/main-client/main-client.component.ts:142]: Get system config must be unauthorized endpoint

File: src/app/services/cordova.service.ts [hunk 2]
```diff
@@ -161,7 +161,7 @@ export class CordovaService {
   }
   playWalkHaveToFinish() {
     if (Capacitor.isNativePlatform() && AudioStatus.walkHaveToFinish) {
-      this.nativeAudio.loop(this.REQUEST_HAVE_TOFINISH);
+      this.nativeAudio.play(this.REQUEST_HAVE_TOFINISH);
```
Comment (heshambadawy) [inline src/app/services/cordova.service.ts:164]: is play sound fixed or must be handled 
?

Comment (Aychihuahuausa) [reply src/app/services/cordova.service.ts:164]: not fixed ,we have a big on the library itself , i fixed it locally

File: src/app/main-client/main-client.component.ts (hunk 1 above)
Comment (Aychihuahuausa) [reply src/app/main-client/main-client.component.ts:142]: it returns 403

**Pull request merged**